cmd/database/database_backup.go

```diff
@@ -40,5 +40,4 @@ func init() {
 	// Update cmd options
 	dbBackupUpdateCmd.Flags().StringVarP(&name, "name", "n", "", "name of the database backup")
 	dbBackupUpdateCmd.Flags().StringVarP(&schedule, "schedule", "s", "", "schedule of the database backup in the form of cronjob")
-	dbBackupUpdateCmd.Flags().IntVarP(&count, "count", "c", 0, "number of backups to keep")
```

Comment:
is this breaking change? 

How can user configure this now?

Reply:
we support 1 scheduled backup by default from backend

Reply:
And if they want to store more?

Reply:
It's free so we support only one.

Reply:
They can create manual backups which chargeable